### PR TITLE
data: money8203

### PR DESCRIPTION
**Pull request by money8203**
## Changes proposed

Update my profile:
- Added links to certain social platforms
- Added social shortcuts

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2887"></a>

### CODE REVIEW

File: data/money8203.json [hunk 1]
```diff
@@ -1,23 +1,44 @@
...
+    { "icon": "FaTwitter", "url": "https://twitter.com/ManishT93598843" },
+    { "icon": "FaGithub", "url": "https://github.com/money8203" },
+    { "icon": "FaLink", "url": "https://manishtyagi.carrd.co/" }
+  ]
   "links": [
+      {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ManishT93598843",
+      "icon": "FaTwitter"
+    }
```
Comment (ChinmayMhatre) [inline data/money8203.json:17]: Missing comma here

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
I noticed a missing comma which could be why the action is failing

File: data/money8203.json (hunk 1 above)
Comment (money8203) [reply data/money8203.json:17]: Ooooooh, thanks a lot Chinmay!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**